Add generate diff command to create html difference reports (#199)
gardener/diki: README.md

#### Difference

> Diki can generate a json containing the difference between 2 output files of `diki run` executions. This can help to identify improvements (or regressions). A human readable html difference report can also be generated from the json difference reports.

Reviewer: ```suggestion
Diki can generate a json containing the difference between 2 output files of `diki run` executions. This can help to identify improvements (or regressions). A human readable html difference report can be generated from the difference reports.
```

- Generate json difference between 2 reports
```bash
diki report diff --title=Title --old=output1.json --new=output2.json --output=difference.json
```

> - Generate html difference report from 1 or more json difference reports.

Reviewer: ```suggestion
- Combine one or more json difference reports to an html report.
```

```bash

> diki report generate diff --unique-attributes=gardener=id --output=difference.html difference1.json difference2.json

Reviewer: ```suggestion
diki report generate diff --identity-attributes=gardener=id --output=difference.html difference1.json difference2.json
```
